frontend/src/components/Consensus/ConsensusBlock.tsx

```diff
@@ -125,9 +125,7 @@ export class ConsensusBlock extends React.Component<ConsensusBlock.Props, {}> {
                   className="legend"
                   key={'block_row_' + this.props.height + '_legend'}
                 >
-                  <Tooltip text={`Block number: ${this.props.height}`}>
-                    {this.displayBlockNumber()}
-                  </Tooltip>
+                  <Tooltip text={`Block number: ${this.props.height}`} />
```

Comment:
This component is unused currently, so I guess that's why we never found this oversight :).

The child should be moved adjacent to `Tooltip`:

```suggestion
                  <Tooltip text={`Block number: ${this.props.height}`} />
                  {this.displayBlockNumber()}
```

Need to restore the method obviously :)

Reply:
oh, I see! thank you for your comment!